packages/pds/src/app-view/subscription/repo.ts

```diff
@@ -27,6 +31,13 @@ export class RepoSubscription {
         const { ran } = await this.leader.run(async ({ signal }) => {
           const sub = this.getSubscription({ signal })
           for await (const msg of sub) {
+            if (!isCommit(msg)) {
+              appViewLogger.warn(
+                { msg },
+                'unexpected message on repo subscription stream',
+              )
+              continue
```

Comment:
Arguably we should move the cursor forward in this situation.  But not terribly worried about it, since this area will get more attention soon.

Reply:
yeah you're right that we should ideally. but I think I'll hold off on it until we start actually sending/processing those messages. there's not any way for this to currently get into a bad state